index.html

```diff
@@ -1349,6 +1347,11 @@ <h1>Glossary</h1>
                 <dd>The bounding box of an input (most commonly, touch) on a digitizer. This typically refers to devices with coarser pointer input resolution than a single pixel. Some devices do not report this data at all.</dd>
             <dt><dfn>digitizer</dfn></dt>
                 <dd>A type of input sensing device in which a surface can detect input which is in contact and/or in close proximity. Most commonly, this is the surface that senses input from the touch contact or a pen stylus.</dd>
+            <dt><dfn>direct manipulation</dfn></dt>
+                <dd>Certain user agents (such as browsers on a touchscreen device) implement a "direct manipulation" metaphor where a pointer not only interacts with controls, but
+                is also used to directly pan or zoom the current page, providing the illusion of direct physical contact. As an example, users on a touchscreen device are generally
+                able to use a finger or a stylus to "grab" a page and pan it by moving the pointer, directly manipulating the page. Contrast this with a mouse pointer on a regular
+                desktop/laptop, where panning is done by using a scrollbar, rather than by "dragging" the page.</dd>
```

Comment:
Note that on many laptops the touchpad provides a bit of a mixed experience, where you are physically manipulating (indirectly) the page, in the same way that an external stylus is also indirectly manipulating the page. The difference here seems to be that the touchpad touches are not sent directly to the page but produce actions through the mouse pointer.

TLDR; it may be worth mentioning that touchpads are similarly indirect, perhaps by pointing out that they only serve to produce actions through the indirect mouse pointer?

Reply:
ah yeah, i believe touchpads fake scroll wheel events, if i'm not misremembering. yeah, will try and put in some wording to that effect, good catch

Reply:
I think in PE world, touchpad is mouse because it "shares" the mouse cursor on screen and appears as (pointerId=mouseId, pointerType="mouse").  Wheel events from trackpad form a completely separate event stream---same as a keyboard scroll---the mouse cursor position remains unchanged and doesn't affect any PointerEvent stream.  I mean we don't `pointercancel` any pointerId (eventTarget changes though).